CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * [#5987](https://github.com/rubocop-hq/rubocop/issues/5987): Suppress errors when using ERB template in Rails/BulkChangeTable. ([@wata727][])
 * [#5966](https://github.com/bbatsov/rubocop/issues/5966): Fix a false positive for `Layout/ClosingHeredocIndentation` when heredoc content is outdented compared to the closing. ([@koic][])
+* Fix auto-correct support checker on --auto-gen-config. ([@r7kamura][])
```

Comment:
Can you extend this to mention that this is related to custom cops and cop extensions? Right now the entry is a bit vague.

Reply:
Good point 👍